### ISSUE TITLE
Consider getting rid of the BulmaClassName type

### ISSUE THREAD

**Issue by toolness** [JustFixNYC/tenants2#321]
I was skeptical about it when I originally wrote it, though I still like the general idea, just because it helps ensure that we don't make typos, and can also (theoretically) help us analyze our class usage for things like [critical styles](https://www.smashingmagazine.com/2015/08/understanding-critical-css/).

However, we're using regular ol' `className` in so many places that the benefits aren't even fully realized, and `className={bulmaClasses("a", "b", "c")}` is much harder to read and write than just `className="a b c"` regardless.

Also, I had a silly dream that perhaps we could have some process whereby our TS automatically knew what all our CSS classes were (via something similar to our querybuilder for GraphQL), but actually realizing that is hard, and the stopgap measure of manually adding every Bulma class we use to the `BulmaClassName` type is cumbersome.

And then there's funky issues like the one run into by @romeboards in https://github.com/JustFixNYC/tenants2/pull/318, that just make it harder for newcomers to contribute to the codebase.

In other words, BulmaClassName just adds a lot of complexity without a ton of benefit, so we should consider getting rid of it.
